packages/graphiql/src/components/DocExplorer/SearchBox.js

```diff
@@ -26,16 +26,21 @@ export default class SearchBox extends React.Component {
   render() {
     return (
       <label className="search-box">
+        <div className="search-box-icon" aria-hidden="true">{'\u26b2'}</div>
         <input
           value={this.state.value}
           onChange={this.handleChange}
           type="text"
           placeholder={this.props.placeholder}
+          aria-label={this.props.placeholder}
```

Comment:
Why use the placeholder here? Shouldn't it be `Search Input`?

Reply:
It really shouldn't be the _placeholder_, that's for sure. Placeholders are no replacement for proper labels though they are quite often (mis)used as such, which is what I was guessing was the case here. Seeing how the placeholder was therefore really used as a label, I wanted to make sure that this label was communicated to assistive technology. 

While the `placeholder` attribute itself _should_ be used when computing the accessible name exposed to assistive technology, this functionality was only introduced recently and is still not widely supported: https://github.com/w3c/html-aam/issues/167. I therefore opted to expose the placeholder explicitly through an `aria-label`.

A better way to go about this would be to add a proper label to the search input, though this would require a slightly more involved change to the design of the interface.